<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorForm</class>
 <widget class="QWidget" name="CalculatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inputSpinBox1"/>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2_2_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outputWidget">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignAbsolute|Qt::AlignBaseline|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3_2">
     <property name="text">
      <string>=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inputSpinBox2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
